definitions/npm/react-redux_v4.x.x/flow_v0.30.x-/react-redux_v4.x.x.js

```diff
@@ -1,6 +1,6 @@
 /* @flow */
-type ConnectAll = <D, P, S, C: React$Component<D, P, S>, SP, DP, Dispatch: Function>(
-  mapStateToProps: (state: Object, ownProps: $Diff<$Diff<$Diff<P, DP>, SP>, D>) => SP,
+type ConnectAll = <D, P, S, C: React$Component<D, P, S>, SP, DP, Dispatch: Function, NP:$Diff<P, {dispatch: Dispatch} & SP & D>>(
```

Comment:
In this case you don't want to add `{dispatch: Dispatch}`.

Since we have an actual `dispatchToProps` argument, dispatch itself is not injected.